specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json

```diff
@@ -424,6 +424,48 @@
           "type": "string"
         }
       }
+    },
+    "StorageTaskQueuedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Storage.StorageTaskQueued event.",
+      "type": "object",
+      "properties": {
+        "queuedDateTime": {
+          "description": "The time at which a storage task was queued.",
+          "format": "date-time",
+          "type": "string"
+        },
+        "taskExecutionId": {
+          "description": "The execution id for a storage task.",
+          "type": "string"
+        }
+      }
+    },
+    "StorageTaskCompletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Storage.StorageTaskCompleted event.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "The status for a storage task.",
+          "type": "string"
+        },
+        "completedDateTime": {
+          "description": "The time at which a storage task was completed.",
+          "format": "date-time",
+          "type": "string"
+        },
+        "taskExecutionId": {
+          "description": "The execution id for a storage task.",
+          "type": "string"
+        },
+        "taskName": {
+          "description": "The task name for a storage task.",
+          "type": "string"
+        },
+        "summaryReportBlobUrl": {
+          "description": "The summary report blob url for a storage task",
+          "type": "string"
```

Comment:
"format": "url"
x-ms-client-name using "Uri" is .NET specific

Reply:
> "format": "url"

updated